src/MigrationTools.Clients.TfsObjectModel/Processors/TfsWorkItemMigrationProcessor.cs

```diff
@@ -874,6 +874,7 @@ private WorkItemData ReplayRevisions(List<RevisionItem> revisionsToMigrate, Work
                         targetWorkItem.ToWorkItem().History = history.ToString();
                     }
                     targetWorkItem.SaveToAzureDevOps();
+                    PatchClosedDate(sourceWorkItem, targetWorkItem);
```

Comment:
This call needs the rest API and needs to be wrapped in a ProductVersion check for not being OnPremisesClassic

Reply:
We support TFS 2013 which has no rest API's at all.